test_util/integration_test.py

```diff
@@ -1340,3 +1349,148 @@ def wait_for_endpoint():
     r_data['Properties']['clusterId'] = ''
 
     assert r_data == exp_data
+
+
+class AgentManipulator:
+
+    def __init__(self, cluster, ssh_user, key_path):
```

Comment:
Add some docs to the init method for clarification (`cluster` is a bit too broad for end-user to know what to pass here).